packages/fxa-auth-server/lib/luaScripts/getSessionTokens.lua

```diff
@@ -0,0 +1,52 @@
+local REDIS_SESSION_TOKEN_PROPERTIES = {
+  'lastAccessTime',
+  'location',
+  'uaBrowser',
+  'uaBrowserVersion',
+  'uaOS',
+  'uaOSVersion',
+  'uaDeviceType',
+  'uaFormFactor'
+}
+local REDIS_SESSION_TOKEN_LOCATION_PROPERTIES = {
+  'city',
+  'state',
+  'stateCode',
+  'country',
+  'countryCode'
+}
+
+local function unpackArray(keys, values)
```

Comment:
I'm not normally one to ask to document "how" things work, but I'm a redis-lua n00b and have no idea which is what and what is that. Some brief documentation about what's happening here would help other newcomers too. Are unpack and unpackArray just deserializing the result that's stored in redis? If so, how come it's not possible to use cjson.decode(value) directly?